testslide/mock_callable.py

```diff
@@ -421,6 +421,18 @@ def _patch(target, method, new_value):
         original_callable = None
     else:
         original_callable = getattr(target, method)
+        if not callable(original_callable):
+            raise ValueError(
+                "mock_callable() can only be used with callable attributes and {} is not.".format(
+                    repr(original_callable)
+                )
+            )
+        if type(original_callable) is type:
```

Comment:
Does this work right for python 2? The error message suggests that we're trying to reject classes, but old-style python2 classes are still allowed:

```
>>> class foo(object):
...     pass
...
>>> type(foo) is type
True
```

```
>>> class bar:
...     pass
...
>>> type(bar) is type
False
```